tests/networks/pop.toml

```diff
@@ -24,4 +24,16 @@ default_command = "pop-node"
 
 [[parachains.collators]]
 name = "pop"
-args = ["-lruntime::contracts=debug"]
\ No newline at end of file
+args = ["-lruntime::contracts=debug"]
+
+[[hrmp_channels]]
```

Comment:
Has channels for testing. Without this PR, this network config should result in no parachain blocks produced.